src/components/MainNav/MainNav.test.js

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import { MainNav } from './MainNav';
+
+test('Main Navigation renders successfully', () => {
+    render(
+        <BrowserRouter>
+            <MainNav />
+        </BrowserRouter>
+    );
+    const homeLinkElement = screen.getByText(/Home/i);
+    expect(homeLinkElement).toBeInTheDocument();
+    const aboutLinkElement = screen.getByText(/Counter/i);
+    expect(aboutLinkElement).toBeInTheDocument();
+});
```

Comment:
If you feel like testing these (honestly up to you if you think you'll get something useful from it) check out the fireEvent object in RTL:
`import { fireEvent, render, screen } from "@testing-library/react";`
`fireEvent.click(screen.getByTestId("testElement")`